dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -126,6 +126,21 @@ def _check_iteration_stop(self):
                 self.reset()
             raise StopIteration
 
+
+def non_empty(shape):
+    for s in shape:
+        if s == 0:
+            return False
+
+    return True
```

Comment:
```suggestion
return all(shape)
```
or
```suggestion
return all(x != 0 for x in shape)
```
But that's up to you. This one is perfectly readable.